### ISSUE TITLE
fix(foundryup): user manually had to add to .bashrc

### ISSUE THREAD

**Issue by transmissions11** [gakonst/foundry#502]
still getting diagnostic info but tracking here for my own sake: https://twitter.com/0xModene/status/1483491515896840192?s=20

**Comment by transmissions11** [comment]
seems like a non issue, just a personal preference by them, followed up

**Comment by kargakis** [comment]
Yeah, I think adding to `.bashrc` is better than just `.profile`. I am on a Debian distro, using stock setup, and still have to manually source `.profile` every time I open up a new terminal.

**Comment by transmissions11** [comment]
alright #539 should close
